sfja/src/components/Students/Students.js

```diff
@@ -220,26 +252,35 @@ class Students extends React.Component {
                     </TableRow>
                   </TableHead>
                   <TableBody>
-                    {students.map((student) => (
-                      <TableRow key={student.student_id}>
-                        <TableCell component="th" scope="row"
-                          className={tableStyle}>
-                          <NavLink to={'/profile/' + student.student_id}>
-                            <Typography align="center" className={tableStyle}>
-                              {student.first_name}</Typography>
-                          </NavLink>
-                        </TableCell>
-                        <TableCell align="center" className={tableStyle}>
-                          {student.last_name}</TableCell>
-                        <TableCell align="center" className={tableStyle}>
-                          {student.grade}</TableCell>
-                        <TableCell align="center" className={tableStyle}>
-                          {student.DOB}</TableCell>
-                        <TableCell align="center" className={tableStyle}>
-                          {student.forms_completed}
-                        </TableCell>
-                      </TableRow>
-                    ))}
+                    {students.map((student) => {
+                      // only supporting grade filtering now
+                      if (filters.grades[student.grade]) {
```

Comment:
Here is where we do filtration. Left it like this now but in the future if we want to add more filters, other code is modular enough to handle, except for this part